util/venv-dependencies.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+set -e
+
+# quiet=--quiet
+quiet=
+
+#### Options end here.
+
+willexit=
```

Comment:
What is `willexit` being used for? Why not `exit 1` in place instead? It seems clearer unless I'm missing a detail.

Reply:
Well, it clearly needs a comment, or at least some renaming. 

My intent: if somebody just runs `util/venv-dependencies.sh` with no arguments, tell them usage; if somebody doesn't have pip, tell them to get it; if they have neither *tell them both things*.

Reply:
Ahhh! I missed the "tell them both things" case :+1: - this makes sense to me now.